mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -302,6 +301,70 @@ resource "google_project_iam_member" "composer-worker" {
 }
 ```
 
+### Cloud Composer 3 networking configuration
+
+In Cloud Composer 3, networking configuration is simplified compared to
+previous versions. You don't need to specify network ranges, and can attach
+custom VPC networks to your envrionment.
+
+<Note>
```

Comment:
I'm pretty sure this HTML element doesn't do anything useful - you might want to rewrite this to use supported callouts instead: https://googlecloudplatform.github.io/magic-modules/develop/handwritten-docs-style-guide/#callouts. You can preview the results in Hashicorp's docs preview tool: https://registry.terraform.io/tools/doc-preview

Reply:
fixed the callouts